Client/Frontend/Browser/TabManager.swift

```diff
@@ -354,13 +354,14 @@ class TabManager : NSObject {
         assert(NSThread.isMainThread())
         // If the removed tab was selected, find the new tab to select.
         if tab === selectedTab {
-            if let index = getIndex(tab) {
-                if index + 1 < count {
-                    selectTab(tabs[index + 1])
+            let viableTabs: [Tab] = tab.isPrivate ? privateTabs : normalTabs
```

Comment:
Might be useful if this were moved to a computed property (e.g., `currentTabs`). I think we do this same calculation in a number of other places.